src/tools/rust-analyzer/crates/proc-macro-srv/src/abis/abi_sysroot/rustc_server.rs

```diff
@@ -288,7 +288,7 @@ impl server::FreeFunctions for Rustc {
         &mut self,
         _s: &str,
     ) -> Result<bridge::Literal<Self::Span, Self::Symbol>, ()> {
-        todo!("implement literal_from_str")
+        panic!("FIXME: implement literal_from_str")
```

Comment:
this specifically shouldn't need to change I think, only the comments

Reply:
RA has a tidy check that caught this `todo!`.

I've switched strategies: I'm going to implement all that in the RA repository first (with `cargo +stage1 test --features rust-analyzer/in-tree`) - there's been too many bridge changes to just tackle them in this monster PR.